.github/workflows/test.yml

```diff
@@ -9,25 +9,41 @@ on:
 
 jobs:
   test-template:
-    runs-on: ubuntu-latest
+    name: ${{ matrix.os }} py${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
```

Comment:
I'm testing on other OS because I was recommended to use `shlex` - which broke stuff on Windows. So now testing.